# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/hddtemp-no_backtrace.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- hddtemp-0.3-beta12/src/backtrace.c.orig	2005-01-27 10:26:08.000000000 -0800
+++ hddtemp-0.3-beta12/src/backtrace.c	2005-01-27 10:26:52.000000000 -0800
@@ -19,7 +19,7 @@
 
 #include <features.h>
 
-#if defined(__i386__) && defined(__GLIBC__)
+#if 0 && defined(__i386__) && defined(__GLIBC__)
 
 #include <stdio.h>
 #include <stdlib.h>
